Bindings/Packages/com.virtualmaker.bindings/Editor/RangeAttributeDrawer.cs

```diff
@@ -10,7 +10,7 @@ public class RangeAttributeDrawer : PropertyDrawer
         public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
         {
             if (attribute is not RangeAttribute rangeAttribute) { return; }
-            var valueProperty = property.FindPropertyRelative("_value") ?? property;
+            var valueProperty = property.FindPropertyRelative("_value") ?? property.FindPropertyRelative("m_InternalValue") ?? property;
```

Comment:
can you add a comment that `m_InternalValue` is for NetworkProperty?

Reply:
sure

Reply:
```suggestion
            var valueProperty = property.FindPropertyRelative("_value") ?? // vanilla property backing field
                property.FindPropertyRelative("m_InternalValue") ?? // network property backing field
                property; // vanilla property
```